src/lang-en.yml

```diff
@@ -0,0 +1,41 @@
+translation:
+  config.add_child: Add Child
+  config.add_spouse: Add Spouse
+  config.add_tree: Add Family
```

Comment:
Use camelCase for phrase. Snake_case is to define context.

Idem for keys below